src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__imac card__imac--bottom"
+        src="/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h2 class="card__apple">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__product card__product--top">Product code: 195434</p>
+
+      <div class="card__review card__review--top">
+        <div class="star stars--4">
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+          <div class="star__stars"></div>
+        </div>
+
+        <p class="card__reviews-value">Reviews: 5</p>
+      </div>
+
+      <div class="card__price card__price--top">
+        <p class="card__price-title">Price:</p>
+
+        <h1 class="card__price-value">$2,199</h1>
+      </div>
+
+      <a
+        href="#"
+        class="card__button card__button--top"
+        data-qa="hover"
+      >
+        BUY
```

Comment:
It's not an issue but I think it's better to add text-transform: uppercase, instead of writing it in actual uppercase in html